packages/jasmine-single/README.md

```diff
@@ -121,16 +130,39 @@ You should add `"node_modules"` like this -
 }
 ```
 
-or else it won't find `@hirez_io/jasmine-single` global types.
-### ⚠ **VS CODE USERS:** 
+### ⚠ **VS CODE USERS:**
 
-Add the above configuration (`types` and/or `typeRoots`) to your `tsconfig.json` specifically or else it would not recognize the global types.
+Adding the above configuration (`types` and/or `typeRoots`) to your `tsconfig.json` (or `tsconfig.spec.json`) is necessary when using VS Code, otherwise the IDE will not recognize the global types.
 
 </details>
 
 <br/>
 
-### Using karma?
+## Configuring Nx
+
+<details>
+<summary>⚠ <strong>CLICK HERE TO EXPAND</strong></summary>
+
+<br/>
+
+When configuring `@hirez_io/jasmine-single` for Nx follow the TypeScript setup provided, and when setting the `types` property under `compileOptions` in `tsconfig.spec.json` apply the same updates to `tsconfig.editor.json` if it exists:
+
+```js
+// For tsconfig.spec.json AND tsconfig.editor.json:
+{
+  "compilerOptions": {
+    "types": [
```

Comment:
@mtpultz thanks!

There's a typo in the code - it should be "jasmine" no?

Plus - have you checked that this configuration works with an nx project that is generated with "jasmine + karma" instead of jest?